hoomd/pytest/test_dcd.py

```diff
@@ -0,0 +1,66 @@
+import hoomd
+import pytest
+import numpy as np
+
+
+def test_attach(simulation_factory, two_particle_snapshot_factory, tmp_path):
+    d = tmp_path / "sub"
+    d.mkdir()
+    filename = d / "temporary_test_file.dcd"
+    sim = simulation_factory(two_particle_snapshot_factory())
+    dcd_dump = hoomd.write.DCD(filename, hoomd.trigger.Periodic(1))
+    sim.operations.add(dcd_dump)
+    sim.operations._schedule()
+    for i in range(10):
+        sim.run(1)
```

Comment:
```suggestion
    sim.run(10)
```
Scheduling is automatic at the first call to `run`.